src/app/dashboard/dashboard.component.html

```diff
@@ -15,14 +15,26 @@
 <div class="flex-container">
   <div class="start-registration">
     <div>Rozpocznij rejetrowanie pracy</div>
-    <button class="clock-in-button">
+    <button (click)="clockIn()" class="clock-in-button">
       <i class="material-icons">access_time</i>
     </button>
+    <ng-container
+      *ngIf="errorClockIn === null; else afterClockInErrorClick"
+    ></ng-container>
+    <ng-template #afterClockInErrorClick>
+      <p>Nie zarejestrować wejścia do pracy Error: {{ errorClockIn }}</p>
+    </ng-template>
```

Comment:
... tutaj dodać warunek, który mówi nam o tym, że chcemy wyświetlać ten fragment w przypadku, kiedy pole errorClockIn nie jest nullem? Czyli dokładnie to co mamy wyżej, tylko o kilka linijek kodu oraz jeden warunek mniej. Dodatkowo nie potrzebujemy ng-template (on i tak jest usuwany z DOMu), warunek możemy nanieść na paragraf.

```suggestion
      <p *ngIf="errorClockIn">Nie zarejestrować wejścia do pracy Error: {{ errorClockIn }}</p>
```